lightwood/api/predictor.py

```diff
@@ -136,13 +136,71 @@ def predict(self, data: pd.DataFrame, args: Dict[str, object] = {}) -> pd.DataFr
         """  # noqa
         pass
 
-    def save(self, file_path: str) -> None:
+
+
+        """
+            NOTE FROM CONTRIBUTOR - will remove before finalizing
+
+            I spotted this code snippet in the custom_cleaner.ipynb file
+            I figured we could maybe have the user pass it optionally
```

Comment:
Let's go for an entirely method `export()`. Changing the `save()` method at this point in time is something we'd rather avoid, even if new arguments are optional.